_includes/earlier_signatures.html

```diff
@@ -1,129 +1,10 @@
-have been victims in our lifetimes.
-
-<p>
-Today we stand together to say: not on our watch, and never again.
-
-<p>
-We commit to the following actions:
-
-<ul>
-  <li>We refuse to participate in the creation of
-  databases of identifying information
-  for the United States government
-  to target individuals based on race, religion, or national origin.
-
-  <li>We will advocate within our organizations:
-    <ul>
-      <li>to minimize the collection and retention of data
-      that would facilitate ethnic or religious targeting.
-      <li>to scale back existing datasets
-      with unnecessary racial, ethnic, and national origin data.
-      <li>to responsibly destroy high-risk datasets and backups.
-      <li>to implement security and privacy best practices,
-      in particular,
-      for end-to-end encryption to be the default wherever possible.
-      <li>to demand appropriate legal process
-      should the government request that we turn over
-      user data collected by our organization, even in small amounts.
-    </ul>
-
-  <li>If we discover misuse of data that we consider illegal or unethical
-  in our organizations:
-    <ul>
-      <li>We will work with our colleagues and leaders to correct it.
-      <li>If we cannot stop these practices,
-      we will exercise our <a href="https://www.shrm.org/resourcesandtools/legal-and-compliance/state-and-local-updates/pages/california-whistleblowers.aspx">rights</a> and responsibilities to speak out publicly
-      and engage in <a href="https://www.theengineroom.org/responsible-data-leaks-and-whistleblowing/">responsible whistleblowing</a>
-      without endangering users.
-      <li>If we have the authority to do so,
-      we will use all available legal defenses to stop these practices.
-      <li>If we do not have such authority,
-      and our organizations force us to engage in such misuse,
-      we will resign from our positions rather than comply.
-    </ul>
-
-  <li>We will raise awareness and ask critical questions
-  about the responsible and fair use of data and algorithms
-  beyond our organization and our industry.
-</ul>
-
-<div class="note">
-Note: Signatories&rsquo; references to affiliated organizations below
-are for identification purposes
-and are not intended to imply an endorsement by the organization.
-</div>
-
-<p>Signed,</p>
 
 <ul class="signatories" id="signatories">
 
 <!-- For lines below not marked with GitHub usernames, see attribution at: -->
 <!-- https://github.com/neveragaindottech/neveragaindottech.github.io/blame/master/index.html -->
 
-<!-- To add a signature, insert ONE LINE below.                            -->
-<!--   - Begin with <li>.                                                  -->
-<!--   - Alphabetize by first name.                                        -->
-<!--   - Consider including your GitHub username (see examples below).     -->
-
+<!-- Please don't add new signatures to this file. -->
```

Comment:
I think that the quickest hack to migrate these is to just process the `<li>`'s and generate files in `_signatures` for them. I think it would be pretty easy to do.